doc/en/user/source/community/geofence/cache.rst

```diff
@@ -0,0 +1,48 @@
+.. _geofence_cache:
+
+GeoFence Cache REST
+===================
+
+The Geofence client cache can be queried and invalidated through a REST service.
```

Comment:
We aren't really querying the cache, more querying the cache status. Also, invalidated isn't very user-speak, so perhaps something like: "The Geofence client cache status can be queried, and the cache cleared (invalidated) through a REST service."